misc/multihash/Cargo.toml

```diff
@@ -16,6 +16,8 @@ readme = "README.md"
 documentation = "https://docs.rs/multihash/"
 
 [dependencies]
+blake2 = { version = "0.7", default-features = false }
 sha1 = "0.5"
 sha2 = { version = "0.7", default-features = false }
 tiny-keccak = "1.2"
```

Comment:
Unrelated to this PR, but v. 1.4 gets rid of assembly code which is good for portability. I think we should upgrade it.

Reply:
That sounds great. Would you like the dependency upgrade to land here? Or would you prefer it be broken out into a separate PR?